shell/platform/tizen/flutter_tizen.cc

```diff
@@ -181,6 +181,11 @@ void FlutterDesktopEngineNotifyAppIsDetached(FlutterDesktopEngineRef engine) {
   EngineFromHandle(engine)->lifecycle_channel()->AppIsDetached();
 }
 
+void FlutterDesktopViewDestroy(FlutterDesktopViewRef view_ref) {
+  flutter::FlutterTizenView* view = ViewFromHandle(view_ref);
+  delete view;
```

Comment:
```suggestion
void FlutterDesktopViewDestroy(FlutterDesktopViewRef view) {
  delete view;
```
will also work but I don't really mind.